### PR TITLE
Restructure docker-compose

### PR DESCRIPTION
**Pull request by canhlinh**
To make the code clean and easier to approach. I have done:
- Remove auto-reload.
- Rename `testnet` to `docker-compose` as the original name with the same purpose
- Modify `goloop2goloop` to follow the convention in `goloop2moonbeam`

### CODE REVIEW

File: docker-compose/goloop2goloop/Makefile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+build:
+	docker-compose build
+	
+run:
+	docker-compose up
+	
+clean:
+	docker-compose down -v --remove-orphans
```
Comment (trmaphi) [inline docker-compose/goloop2goloop/Makefile:8]: Could we use `docker-compose down -v --remove-orphans --rmi local` to remove the built images?

**Review by trmaphi — COMMENTED** [review]
document for https://github.com/icon-project/btp/blob/icondao/doc/tutorial.md#preparation-1  needs to be update, too

File: docker-compose/goloop2goloop/Makefile (hunk 1 above)
Comment (canhlinh) [reply docker-compose/goloop2goloop/Makefile:8]: @trmaphi If we remove the build image, then we lost the cached docker layer. The next time of building images is quite slow.

**Comment by canhlinh** [comment]
> document for https://github.com/icon-project/btp/blob/icondao/doc/tutorial.md#preparation-1 needs to be updated, too

You are right. I'm considering it.

**Pull request merged**